src/daemon/daemon_config.cpp

```diff
@@ -104,10 +104,22 @@ std::unique_ptr<const mp::DaemonConfig> mp::DaemonConfigBuilder::build()
     auto multiplexing_logger = std::make_shared<mpl::MultiplexingLogger>(std::move(logger));
     mpl::set_logger(multiplexing_logger);
 
+    auto storage_path = qgetenv("MULTIPASS_STORAGE");
+
     if (cache_directory.isEmpty())
-        cache_directory = MP_STDPATHS.writableLocation(StandardPaths::CacheLocation);
+    {
+        if (!storage_path.isEmpty())
+            cache_directory = mp::utils::make_dir(QString::fromUtf8(storage_path), "cache");
```

Comment:
Ah that's useful, I think I should have used that `make_dir` a few times already.